react-example/cypress/integration/inapp/inapp.spec.ts

```diff
@@ -1,11 +1,13 @@
 import * as mockMessages from '../../fixtures/inapp/200.json';
 
+const url = `/api/inApp/web/getMessages*`;
```

Comment:
Opted to keep this as a local variable rather than reach outside of the react-example directory.